Rdmp.Dicom/CommandExecution/ExecuteCommandCreateNewImagingDatasetSuite.cs

```diff
@@ -113,9 +113,11 @@ bool createLoad
         public override void Execute()
         {
             if (DicomSourceType == null)
+            {
                 SetImpossible("You must specify a Type for DicomSourceType");
+                return;
```

Comment:
Ah this is wrong, this will swallow the Exception.... Change it to just throw instead of SetImpossible.  The previous logic was to make a last minute choice about impossibility then call base.Execute to throw the Exception. 

As it stands this code change would now silently swallow in a situation where previously it would throw.